aspnet-same-runtime-version/test.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -euo pipefail
+IFS=$'\n\t'
```

Comment:
Let's consistently use the exact same lines as we have here in all scripts:
```
#!/usr/bin/env bash

set -euo pipefail
IFS=$'\n\t'

```
For scripts that `set -x`, let's add it as a line directly after the `IFS` assignment.